.github/workflows/pages.yaml

```diff
@@ -22,22 +21,17 @@ jobs:
         uses: actions/configure-pages@v5
 
       - name: Install uv
-        uses: astral-sh/setup-uv@v3
-        with:
-          enable-cache: true
-          cache-dependency-glob: pyproject.toml
-          cache-suffix: docs
+        uses: astral-sh/setup-uv@v4
```

Comment:
It's actually faster to run this without using the cache.

Reply:
**[Praise]** Common uv W 🚀